docarray/typing/tensor/abstract_tensor.py

```diff
@@ -121,7 +120,25 @@ def __docarray_validate_shape__(cls, t: T, shape: Tuple[Union[int, str]]) -> T:
         tshape = comp_be.shape(t)
         if tshape == shape:
             return t
-        elif any(isinstance(dim, str) for dim in shape):
+        elif any(isinstance(dim, str) or dim == Ellipsis for dim in shape):
+            ellipsis_occurrences = [
+                pos for pos, dim in enumerate(shape) if dim == Ellipsis
+            ]
+            if ellipsis_occurrences:
+                if len(ellipsis_occurrences) > 1:
+                    raise ValueError(
+                        f'Cannot use Ellipsis (...) more than once for the shape {shape}'
+                    )
+                ellipsis_pos = ellipsis_occurrences[0]
+                dimensions_needed = max(len(tshape) - len(shape) + 1, 1)
```

Comment:
can u add a comment on what is happening here ? I am not sure to understand

Reply:
Can you check now? 

The motivation is not to allow validating shapes like [3, ...] against [3]. That's why I specified adding at least 1 extra dimension.

Reply:
looks good now